doc/Compatible-IDE.md

```diff
@@ -0,0 +1,22 @@
+# Compatible IDEs
+
+Here you'll find the full up-to-date list of tested in Eclipse Che infrastructure IDEs:
+
+| JetBrains IDE/Version   | 2020.1.3 | 2020.1.4 | 2020.2.2 | 2020.2.3 | 2020.2.4 | 2020.3.1 | 2020.3.2 | 2020.3.3 | 2020.3.4 | 2020.3.5 | 2021.1 |
```

Comment:
maybe sort by recent to oldest

2021.1, 2020.3.5, etc so we have checkboxes on the left

Reply:
Sorted